### PR TITLE
Make all news posts use proper templates and use the Jekyll post system

### PR DESCRIPTION
**Pull request by PureTryOut**
A TON of layout code was just copied over with every update post. That's just asking for problems and is what templates exists for.

While at it I formatted all the HTML I saw, formatting was all over the place.

Preferably all new posts are made in Markdown. It supports basically everything needed and if something is lacking still you can always still put HTML in there. I converted the last update post into Markdown as an example.

### CODE REVIEW

File: updates/_posts/2021-04-08-The-April-update.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  Okay, what an Easter! We had a few people get the cute Easter emote from turning in eggs and the easter egg hunts 
+  were super fun to attend! Sadly, people found a small bug where you can find an infinite amount of eggs and 
+  one of the rewards being credits from turning in eggs this caused the Easter event to be shut down early. With this we
+  also had an unplanned early release of an update.
+  <br/>
+  <br/>
+  Also the beginning of the 2.009 progress has started which is our new 
+  core being coded from scratch. The progress will be worked on for the next couple months before any real news 
+  will start being released. The first bit of content will be the f2p content and getting that completed. 
+  But worry not there is a way to keep track of the progress thanks to the new member to the bot team,<img height="20" src="../img/zanik.webp" width="20"/> <a href="../../../site/game_guide/zanik.html"><b style="color:goldenrod">Zanik</b></a>!
```
Comment (dginovker) [inline updates/_posts/2021-04-08-The-April-update.html]: This references `zanik.webp` but it looks like `zanik.webp` was deleted without being replace

**Comment by dginovker** [comment]
http://127.0.0.1:4000/services/m=news/archives/ got dehooked up (see: https://2009scape.org/services/m=news/archives/)

image

Looks good though, definitely a needed improvement :smile:

**Comment by PureTryOut** [comment]
Yeah the url got changed to http://127.0.0.1:4000/updates/. I found the original URL scheme weird (I guess that's copied from the original site?) and the new url scheme makes it easier to use with Jekyll's categories systems.

I'll add a redirect to the new url though.

**Comment by dginovker** [comment]
Yea the original one is authentic, which is why we use it. I'm not against to updating it for a new scheme, but all existing references in the codebase should be updated too

**Comment by PureTryOut** [comment]
I've added redirects to the new url scheme for both http://127.0.0.1:4000/services/m=news/archives/ and http://127.0.0.1:4000/services/m=news/latest.html (the latter is afaik used by Saradomin launcher, but preferably that gets updated to use the new scheme rather than relying on the redirect).

**Comment by PureTryOut** [comment]
Btw I also added a RSS feed which is nice for feed readers or applications to get all news posts and a short description about them. It includes an url to open the full post in the browser.

Honestly the RSS feed is the whole reason I started this PR, I need it for a personal project of mine. I ended up in a rabbit hole though lol :sweat_smile:

**Review by dginovker — APPROVED** [review]
I love RSS too :smile: 

Looks good, I'm gonna merge. One small nitpick is all existing update URLs (i.e. https://2009scape.org/services/m=news/archives/2023-11-11.html) now give a 404. Ideally they would at the very least go to https://2009scape.org/services/m=news/archives/, since I do know of a few external places update links were shared (Discord, Lemmy, Forums, etc)

Overall definitely a net positive improvement though, so many thanks!

**Comment by dginovker** [comment]
@PureTryOut Saradomin Launcher isn't expecting these changes, I gotta revert it

https://forum.2009scape.org/viewtopic.php?p=3515-launcher-crashing-on-start#p3515

**Comment by PureTryOut** [comment]
Ugh :cry:  It fully crashes? What the hell... Uhm ok I'll see if I can make it work with what Saradomin expects...

**Pull request merged**